docs/system/system-metrics.md

```diff
@@ -739,3 +740,17 @@ an `{os}` prefix to split this metric across OSes.
 
 [DocumentStatus]: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.22.0/specification/document-status.md
 [MetricRecommended]: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.22.0/specification/metrics/metric-requirement-level.md#recommended
+
+### Metric: `system.linux.memory.available`
```

Comment:
I think maybe for now is fine to leave this here, but if more OS's specific things pop up, we probably should create a new markdown file for it.

Reply:
acknowledged